src/components/AirtableView.jsx

```
@@ -0,0 +1,381 @@
+    if (!activeView) return null;
+
+    switch (activeView.type) {
+      case 'list':
+        return (
+          <ListView
+            table={activeTable}
+            view={activeView}
+            records={displayRecords}
+          />
+        );
+      case 'grid':
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)